ios/Zentacle/Zentacle.entitlements

```diff
@@ -10,6 +10,9 @@
 	<array>
 		<string>zentacle.app.link</string>
 		<string>zentacle-alternate.app.link</string>
+		<string>applinks:zentacle.com</string>
+		<string>applinks:www.zentacle.com</string>
+		<string>applinks:https://zentacle.com</string>
```

Comment:
If this isn’t working could we try adding it both with and without `applinks:`?

Reply:
Sure, I. think we should deploy to TestFlight and see.